### PR TITLE
69879 Adding --endpoint-url to AWS S3 with arg --aws-s3-endpoint-url …

### PR DESCRIPTION
**Pull request by bogdan3d**
…argument.

### CODE REVIEW

File: cassandra_backup_service.py [hunk 1]
```diff
@@ -246,6 +246,9 @@ def append_cqlsh_args(cmd, args):
     if args.cqlsh_user:
         cmd.append('-p')
         cmd.append(args.cqlsh_pass)
+    if args.endpoint_url:
+        cmd.append('--endpoint-url')
+        cmd.append(args.endpoint_url)
```
Comment (vikmalik) [inline cassandra_backup_service.py:251]: This is not cql argument, please remove it

Comment (bogdan3d) [reply cassandra_backup_service.py]: Removed: https://github.com/onzra/cassandra_backup_service/pull/53/commits/cd0ffd6a4b62050a23a48c13247fc3d57be5ff56

**Review by vikmalik — APPROVED** [review]

File: cassandra_backup_service.py (hunk 1 above)
Comment (vikmalik) [reply cassandra_backup_service.py]: Thanks @bogdan3d ! Could you please merge and tag the release

**Comment by bogdan3d** [comment]
Merged to master and tagged 6.1.0: https://github.com/onzra/cassandra_backup_service/releases/tag/6.1.0

File: cassandra_backup_service.py (hunk 1 above)
Comment (bogdan3d) [reply cassandra_backup_service.py]: @vikmalik done!

**Pull request merged**